ptop.py

```diff
@@ -0,0 +1,115 @@
+import time
+import os
+import argparse
+
+parser = argparse.ArgumentParser(
+    prog="tailf",
+    description="Reads and prints the last 10 lines of a file line by line every 10 seconds",
+)
+
+parser.add_argument("logfile", help="Path to the log file")
+parser.add_argument(
+    "--total-jobs", "-tj", dest="tj", type=int, help="The number of jobs in the order"
+)
+args = parser.parse_args()
+tj = args.tj
+
+
+def clear_console():
+    os.system("cls" if os.name == "nt" else "clear")
+
+
+def parse(input_line: str) -> dict:
+    columns = input_line.split()
+    parsed = {
+        "sec": int(columns[0]),
```

Comment:
```suggestion
        "seq": int(columns[0]),
```

As in "sequence number"! This is a bug introduced by 25fd5082ba1c5e3a0793dd1f77837fd945d2970f